libs/database/collection.ts

```diff
@@ -64,7 +68,7 @@ export class _MixinCollection<T> {
 
   aggregate<ResponseType>(
     pipeline?: object[],
-    options: CollectionAggregationOptions = {allowDiskUse: true}
```

Comment:
I think the options could be more than allow disk use, is there any mongodb interface that we can use for aggregate options?